app/build.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id "io.intrepid.static-analysis" version "1.0-alpha6"
```

Comment:
Will this be updated to 1.0 to match the README of the static-analysis-gradle-plugin once it's merged in?